pkg/typesystem/typesystem.go

```diff
@@ -121,6 +131,7 @@ func Intersection(children ...*openfgav1.Userset) *openfgav1.Userset {
 	}
 }
 
+// Difference calculates the set difference between two Usersets 'base' and 'sub', and returns a new Userset.
```

Comment:
Nit: this function has no "calculation". So to be more consistent with the other docs:

```
// Difference creates new Userset representing the difference between two Usersets 'base' and 'sub'
```